### ISSUE TITLE
Group feeds by person?

### ISSUE THREAD

**Issue by bookish1303** [kickscondor/fraidycat#76]
**Before you start.**

* [x] I have checked the [Rejected Features](https://github.com/kickscondor/fraidycat/wiki/Rejected-Features) page.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**My great idea for Fraidycat is:**
I follow people who have multiple accounts across multiple services. It would be nice to group them all into one sort of "identity," smaller than the tagging functionality already present in the service. If the goal of Fraidycat is to follow "people," rather than just accounts on one service or another, this could be a worthwhile feature. This functionality seems to be intimated by a lot of the press Fraidycat is getting today, so I hope it might be a feature that is considered.

**Comment by kickscondor** [comment]
I could have sworn I already had an issue open for this - but I can't find it, so you win the award for kicking it off! Yes - I am champing at the bit to get to this feature. I've actually designed the whole thing already - and I've got a very exciting way of doing it I think.

In fact, if you look at the data that Fraidycat stores, you'll see that I'm already storing related sites. If a URL is found in the Twitter bio, it will be scanned for additional 'sources'. Same if the website has rel="me" links under the hood. (The key though is that the two links have to link to each other - so if you have markwahlberg.com in your bio, that doesn't mean you're markwahlberg.com - that site would need to link back to you in its tags.)

You'll also be able to group together arbitrary sources into a new identity. They'll all be listed on one edit page.

The minute I'm certain that the new guts of Fraidycat 1.1 are stable, this is going in. Glad to hear at least one other person is into it. ;D
